From 6e2a0693206573b081a79b9dd37c1ab59273549d Mon Sep 17 00:00:00 2001
From: Mooffie <mooffie@gmail.com>
Date: Wed, 1 Jul 2015 20:37:42 +0300
Subject: [PATCH] Ticket #3493: menu selectors: make "User", not "Local", the
 focused button.

On menu selectors two buttons are shown. Users are more likely to
want to edit their central user menu, so we move the focus there.
---
 src/editor/editcmd.c  | 1 +
 src/filemanager/cmd.c | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/editor/editcmd.c b/src/editor/editcmd.c
index ea9f49f..a2f2252 100644
--- a/src/editor/editcmd.c
+++ b/src/editor/editcmd.c
@@ -2170,6 +2170,7 @@ edit_load_menu_file (WDialog * h)
     int dir;
     gboolean ret;
 
+    query_set_sel (1);
     dir = query_dialog (_("Menu edit"),
                         _("Which menu file do you want to edit?"), D_NORMAL,
                         geteuid () != 0 ? 2 : 3, _("&Local"), _("&User"), _("&System wide"));
diff --git a/src/filemanager/cmd.c b/src/filemanager/cmd.c
index d824c3f..7a8fa8b 100644
--- a/src/filemanager/cmd.c
+++ b/src/filemanager/cmd.c
@@ -1011,6 +1011,7 @@ edit_mc_menu_cmd (void)
     vfs_path_t *menufile_vpath;
     int dir = 0;
 
+    query_set_sel (1);
     dir = query_dialog (_("Menu edit"),
                         _("Which menu file do you want to edit?"),
                         D_NORMAL, geteuid ()? 2 : 3, _("&Local"), _("&User"), _("&System Wide"));
-- 
2.1.4

